src/hooks/Projects.ts

```diff
@@ -125,6 +129,64 @@ export function useProjectsSearch(handle: string | undefined) {
   )
 }
 
+export function useTrendingProjects() {
+  const secondsInDay = 24 * 60 * 60
+
+  const payments = useSubgraphQuery(
+    {
+      first: 1000,
+      entity: 'payEvent',
+      keys: [
+        'amount',
+        {
+          entity: 'project',
+          keys: ['id'],
+        },
+      ],
+      where: [
+        {
+          key: 'timestamp',
+          value: parseInt(
+            (
+              new Date().valueOf() / 1000 -
+              secondsInDay * trendingWindowDays
+            ).toString(),
+          ),
+          operator: 'gte',
+        },
+      ],
+    },
+    {
+      staleTime: 10800000, // 3 hours
+    },
+  )
+
```

Comment:
**question**: wondering if there's a way we can bulk-request a list of project IDs, like
`Where projectID in [...]`

Reply:
i've been poking at this, but haven't figured it out yet. if we could, it'll reduce the needed queries down to 2!